decidim-initiatives/app/views/layouts/decidim/admin/initiative.html.erb

```diff
@@ -20,7 +20,7 @@
           <ul>
             <% current_participatory_space.components.each do |component| %>
               <% if component.manifest.admin_engine %>
-                <li <% if is_active_link?(manage_component_path(component)) || participatory_space_active_link?(component) %> class="is-active" <% end %>>
+                <li <% if is_active_link?(manage_component_path(component)) %> class="is-active" <% end %>>
```

Comment:
is this a bugfix? why this modification in the initiatives module? is seems non related..

Reply:
Yeah, this was previously implemented when adding `trustees` to all `participatory_spaces` that have an `election` module. But `initatives` don't have an `election`, so I removed the link for `initiatives` again.